app/views/books/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1>Title</h1>
+  <%= @book.title %>
+  <h1>ISBN</h1>
+  <%= @book.isbn %>
+  <h1>Year of Published</h1>
+  <%= @book.year %>
+  <h1>Author</h1>
+  <%= @book.author.first_name + ' ' + @book.author.last_name %>
```

Comment:
We prefer string interpolation since this is safer than concatenation.
http://stackoverflow.com/questions/6778192/rails-string-interpolation-in-a-string-from-a-database